docs/source/user-guide/mission-modeler-guide/activity-types/effect-model.md

```diff
@@ -0,0 +1,128 @@
+<!-- - `waitForChildren()`: Delay the currently-running activity until all child activities have completed. On resumption, it will observe effects caused by other activities over the intervening timespan. -->
+
+Actions related to spawning other activities are provided by the generated `ActivityActions` class, usually found under the `generated` package within your codebase.
+
+- `spawn(activity)`: Spawn a new activity as a child of the currently-running activity at the current point in time. The child will initially see any effects caused by its parent up to this point. The parent will continue execution uninterrupted, and will not initially see any effects caused by its child.
+<!-- - `spawnAfter(duration, activity)`: Asynchronously spawn a new activity as a child of the currently-running activity at the specified duration after the current point in time. The child will initially see any effects caused by its parent up to that point. The parent will continue execution from the current time point uninterrupted, and will not initially see any effects caused by its child. -->
+- `call(activity)`: Spawn a new activity as a child of the currently-running activity at the current point in time. The child will initially see any effects caused by its parent up to this point. The parent will halt execution until the child activity has completed. This is equivalent to calling `waitFor(spawn(activity))`.
+
+
+For example, consider a simple activity for running the on-board heaters:
+
+```java
+@ActivityType("RunHeater")
+public final class RunHeater {
+    private static final int energyConsumptionRate = 1000;
+
+    @Parameter
+    public long durationInSeconds;
+
+    @Validation("duration must be positive")
+    @Validation.Subject("durationInSeconds")
+    public boolean validateDuration() {
+        return durationInSeconds > 0;
+    }
+
+    @EffectModel
+    public void run(final Mission mission) {
+        spawn(new PowerOnHeater());
+
+        final double totalEnergyUsed = durationInSeconds * energyConsumptionRate;
+        mission.batteryCapacity.use(totalEnergyUsed);
+
+        delay(durationInSeconds, Duration.SECONDS);
+
+        call(new PowerOffHeater());
+    }
+}
+```
+
+This activity first spawns a `PowerOnHeater` activity, which then continues concurrently with the current `RunHeater` activity. Next, the total energy to be used by the heater is subtracted is subtracted from the remaining battery capacity (see [Models and Resources](https://github.com/NASA-AMMOS/aerie/wiki/Models-and-Resources)). The energy used depends on the duration parameter of the activity, allowing the activity's effect to be tuned by the planner. Next, the activity waits for the desired heater runtime to elapse, then spawns and waits for a `PowerOffHeater` activity. The activity completes after both children have completed.
```

Comment:
Link needs to be updated: `(see [Models and Resources](https://github.com/NASA-AMMOS/aerie/wiki/Models-and-Resources))`